lightning/src/ln/channel.rs

```diff
@@ -2130,7 +2153,11 @@ impl<Signer: Sign> Channel<Signer> {
 		} else if their_features.supports_channel_type() {
```

Comment:
why not just `else if !their_features.supports_channel_type() {`, removing the empty block?